pyproject.toml

```diff
@@ -0,0 +1,74 @@
+[build-system]
+requires = ["flit_core >=3.2,<4"]
+build-backend = "flit_core.buildapi"
+
+[tool.flit.module]
+name = "cachecontrol"
```

Comment:
I went with `flit` because it's what I've used it in the past, but I'm happy to use another build backend (like setuptools' own backend) if you'd prefer.

Reply:
I remember `flit build` and `python -m build` produce different distribution files. Can you verify all required files are included in the sdist and wheel by running `python -m build`?

Reply:
The wheel looks correct to me, comparing against [this](https://inspector.pypi.io/project/cachecontrol/0.13.0/packages/65/03/9ed07561cc4b428204fc59fa090ae6652b6fe0a6506c2b05adfcb09085b9/CacheControl-0.13.0-py3-none-any.whl/).

The sdist also looks "correct," (from [this](https://inspector.pypi.io/project/cachecontrol/0.13.0/packages/9d/47/9bdf134012537324bbdebe84c144cc8fab08f148fc6c9ce89b8eefff1cb0/CacheControl-0.13.0.tar.gz/)) although now it's missing the `tests` module. I can try and re-add that if you'd like, although I'm not sure if it's considered idiomatic anymore to include the testsuite in the sdist.

Reply:
I also confirmed that both the wheel and sdist install correctly (via `python -m pip install dist/....`) and pass the local dev copy's tests.

Reply:
Yes, include the `tests` for sdist please.

Reply:
Sure thing, done with f3e4222.